README.md

```diff
@@ -40,90 +40,33 @@ An easy way to get the full Panoptes stack running (see `fig.yml` to dig into th
   * [Ubuntu](https://docs.docker.com/installation/ubuntulinux/) - Docker
   * [Windows](http://docs.docker.com/installation/windows/) - Boot2Docker
 
-#### Installation
+* [fig](http://fig.sh)
 
-1. Ensure your repo directory starts with a lowercase letter - you may need to move `/Panoptes` to `/panoptes`.
+#### Usage
 
-2. Prepare your fig development environment config files. You should only have to do this before the first boot. **Note:** The fig docker environment uses linked docker containers, so your Postgres and Zookeeper hosts urls need to refer to these containers.
+1. Setup the application configuration files by running `rake configure FIG_RAKE=off`
```

Comment:
I think that's fine. I'm going to allow this variable to set from a `.fig_rake` file as well.